exporter/awsexporter/aws_xray.go

```diff
@@ -146,25 +155,12 @@ func transformConfigToXRayOptions(axrCfg *awsXRayConfig) (xopts []xray.Option, e
 
 // ExportSpans is the method that translates OpenCensus-Proto Traces into AWS X-Ray spans.
 // It uniquely maintains
-func (axe *awsXRayExporter) ConsumeTraceData(ctx context.Context, td data.TraceData) (xerr error) {
-	ctx, span := trace.StartSpan(ctx,
-		"opencensus.service.exporter.aws_xray.ExportSpans",
-		trace.WithSampler(trace.NeverSample()))
-
-	defer func() {
-		if xerr != nil && span.IsRecordingEvents() {
-			span.SetStatus(trace.Status{
-				Code:    int32(trace.StatusCodeUnknown),
-				Message: xerr.Error(),
-			})
-		}
-		span.End()
-	}()
-
+func (axe *awsXRayExporter) PushTraceData(ctx context.Context, td data.TraceData) (int, error) {
 	serviceName := td.Node.GetServiceInfo().GetName()
```

Comment:
Is there a way to refactor the recording of `serviceName` into `PushTraceData`? Maybe add a TODO.

Reply:
Do you mean to add it in the exporterhelper decorator?

Reply:
Yes, like `pushTraceDataWithSpan`.

Reply:
DOne.